index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms">
```

Comment:
- [ ] Kindly uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
      - `action` value should be similar to `https://formspree.io/f/{form_id}`
      -  `method` value equals `POST`

Reply:
okay, let me do it well